ducktape/command_line/main.py

```diff
@@ -35,9 +39,23 @@ def parse_args():
                         help='one or more space-delimited strings indicating where to search for tests')
     parser.add_argument("--collect-only", action="store_true", help="display collected tests, but do not run")
     parser.add_argument("--debug", action="store_true", help="pipe more verbose test output to stdout")
+    parser.add_argument("--cluster", action="store", default="ducktape.cluster.vagrant.VagrantCluster",
+                        help="Cluster class to use to allocate nodes for tests")
     parser.add_argument("--exit-first", action="store_true", help="exit after first failure")
 
-    args = parser.parse_args()
+    project_configs = []
+    project_config_file = os.path.join(ConsoleConfig.METADATA_DIR, 'config')
+    if os.path.exists(project_config_file):
+        project_configs = list(itertools.chain(*[line.split() for line in open(project_config_file).readlines()]))
+
+    user_configs = []
+    user_config_file = os.path.expanduser('~/.ducktape/config')
```

Comment:
Whatever convention is chosen for location and name of config, the path/name should probably live in ConsoleConfig in config.py